js/chat_app/src/components/Login/Login.css

```diff
@@ -0,0 +1,8 @@
+.login-form {
```

Comment:
If you're using Material UI, the best way would be to use JSS. Or style components like in MUI docs

Reply:
fixed it, maybe you meant another thing?